### PR TITLE
Add Multisig platform to Showcases & update Mesh SDK description

### PR DESCRIPTION
**Pull request by jinglescode**
Hi,

Adding Multisig platform into showcase, as its a project/product for end users.

Multisig platform is for project teams, a product that allows teams to participant in governance. 
Check [features page](https://multisig.meshjs.dev/features) for an idea, or simply connect your preprod wallet to try it out.

### CODE REVIEW

File: src/data/builder-tools.js [hunk 1]
```diff
@@ -850,7 +850,7 @@ export const Showcases = [
   },
   {
     title: "Mesh SDK",
-    description: "An Open-Source library by Martify Labs with the purpose of making building dApps on Cardano easier than ever. The Mesh library is engineered to simplify the development and implementation of blockchain technology, so you can focus on your project.",
+    description: "A feature-complete, open-source TypeScript SDK and off-chain framework that includes wallet integration, transaction building, a smart contract library, third-party API integration, UI components, and more. It's thoroughly documented, with live demos, great for both learners and experienced developers.",
```
Comment (rphair) [inline src/data/builder-tools.js:853]:
```suggestion
    description: "A feature-complete, open-source TypeScript SDK and off-chain framework including wallet integration, transaction building, a smart contract library, third-party API integration, and UI components: with thorough documentation and live demos for all skill levels.
```
This was one of the longer `description:`'s in our portfolio and the additions have pushed it to be the new maximum & well above the recognised largest grouping around 240 characters.  Can I please suggest this change according to our usual goal of eliminating redundancy & subjectivity?

File: src/data/showcases.js [hunk 2]
```diff
@@ -1202,6 +1202,15 @@ export const Showcases = [
     source: null,
     tags: ["governance", "analytics"],
   },
+  {
+    title: "Multisig Platform",
+    description:
+      "Secure your treasury and participant in governance, as a team with multi-signature.",
+    preview: require("./showcase/mesh-multisig-platform.jpg"),
+    website: "https://multisig.meshjs.dev/",
```
Comment (rphair) [inline src/data/showcases.js:1210]:
```suggestion
    website: "https://multisig.meshjs.dev/features",
```
Using this page instead would immediately give new users & even _prospective_ users a digest of available features... _as well as_ the `Connect:` button which is the only feature on the former page.

**Review by rphair — COMMENTED** [review]
⭐

**Review by rphair — APPROVED** [review]

**Review by rdlrt — APPROVED** [review]

**Comment by rphair** [comment]
ugh, "build error" was my fault (fixed by @jinglescode in https://github.com/cardano-foundation/developer-portal/pull/1355/commits/b1b88e8338ed9cc54b002b6c2478b48d9549550f): `vim` ate my punctuation 😜

**Comment by jinglescode** [comment]
> ugh, "build error" was my fault from [#1355 (comment)](https://github.com/cardano-foundation/developer-portal/pull/1355#discussion_r1842321523) (fixed by @jinglescode in [b1b88e8](https://github.com/cardano-foundation/developer-portal/commit/b1b88e8338ed9cc54b002b6c2478b48d9549550f)): `vim` ate my punctuation 😜

I thought i fixed it?

**Comment by rphair** [comment]
> I thought i fixed it?

you certainly did... I was accepting responsibility for the error.

**Review by fill-the-fill — APPROVED** [review]

**Pull request merged**